<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="on_air_detection">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>기본 화면</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>931</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>610</y>
        <width>911</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_pause">
         <property name="text">
          <string>▶️</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_stop_start">
         <property name="text">
          <string>⏯️</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QSlider" name="Video_bar">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="Total_length">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Current">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="btn_forward">
         <property name="text">
          <string>⏩</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn_prev">
         <property name="text">
          <string>⏪</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>949</x>
        <y>50</y>
        <width>181</width>
        <height>651</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Video_info">
         <property name="text">
          <string>영상 정보</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="Video_info_text"/>
       </item>
       <item>
        <widget class="QLabel" name="Log">
         <property name="text">
          <string>로그 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="Log_text"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Video_upload">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>동영상 업로드</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>821</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>설정 화면</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>521</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Log</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>521</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>로그 기록</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox1">
          <property name="text">
           <string>Off(Save in TXT)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox2">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>뮤트(Mute)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>521</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Graphic</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>521</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>랙 경계선 표시( Set Border Lines)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox3">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox4">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>랙 구분(상,하  Rack Classification Mode)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Video</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>521</width>
         <height>46</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>  위험상황 기준 전후 상황 기록 </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox">
          <property name="currentText">
           <string>±3초</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="placeholderText">
           <string>5</string>
          </property>
          <item>
           <property name="text">
            <string>±3초</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±5초</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±7초</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>  위험상황 기준 전후 상황 기록 </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_2">
          <property name="currentText">
           <string>5초</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="placeholderText">
           <string>5</string>
          </property>
          <item>
           <property name="text">
            <string>5초</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10초</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15초</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>실시간 탐지</string>
     </attribute>
     <widget class="QPushButton" name="btn_start_detection">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>620</y>
        <width>901</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>한컴 고딕</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>탐지 시작</string>
      </property>
     </widget>
     <widget class="QLabel" name="Log_2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>40</y>
        <width>158</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>로그 </string>
      </property>
     </widget>
     <widget class="QListWidget" name="Log_text_2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>70</y>
        <width>158</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="on_air_camera">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>721</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Log_4">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>158</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>실시간 카메라</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
